tests/actions/ReportTest.ts

```diff
@@ -533,6 +533,8 @@ describe('actions/Report', () => {
                         value: {
                             1: REPORT_ACTION,
                         },
+                        // @ts-expect-error -- it's necessary for the test
+                        shouldNotify: true,
```

Comment:
In order to get rid of this suppression (and in the other files too), you can change `emitOnyxUpdate` [here](https://github.com/Expensify/App/blob/main/tests/utils/PusherHelper.ts#L27) to use `OnyxServerUpdate`:

```ts
function emitOnyxUpdate(args: OnyxServerUpdate[]) {
```